gamestonk_terminal/prediction_techniques/knn_view.py

```diff
@@ -151,7 +151,9 @@ def k_nearest_neighbors(other_args: List[str], s_ticker: str, df_stock: pd.DataF
         preds = knn.predict(X_valid.reshape(X_valid.shape[0], X_valid.shape[1]))
         forecast_data = knn.predict(forecast_data_input.reshape(1, -1))
 
-        forecast_data_df = pd.DataFrame(forecast_data.T, index=future_dates)
+        forecast_data_df = pd.DataFrame(
+            [(i > 0) * i for i in forecast_data.T], index=future_dates
+        )
```

Comment:
I would do max(0, i) instead

Reply:
This one is more optimised

<img width="308" alt="Captura de ecrã 2021-05-20, às 23 28 06" src="https://user-images.githubusercontent.com/25267873/119056717-178fe680-b9c3-11eb-8673-c6abab53e97b.png">

Reply:
Can be faster ;)

`[i if i>0 else 0 for i in ...]`

![Screen Shot 2021-05-20 at 6 59 57 PM](https://user-images.githubusercontent.com/18151143/119059136-b43f8d80-b99d-11eb-9610-f300bebc44cf.png)

Reply:
Then, I agree :D